### PR TITLE
Docs: Mark macro examples as code

### LINKED ISSUE #23840: documentation issue for postgresql
**What happened**:

[Macro documentation](https://grafana.com/docs/grafana/latest/features/datasources/postgres/#macros) for postgresql is incorrect.

**What you expected to happen**:

Documentation says

    $__timeGroup(dateColumn,‘5m’, 0)

but actual use requires that the interval does not have quotes, i.e.

    $__timeGroup(dateColumn,5m, 0)

**How to reproduce it (as minimally and precisely as possible)**:

See above

**Anything else we need to know?**:

I haven't tried the other macros but assume it's the same for all of them.

**Environment**:
- Grafana version: grafana/grafana:6.7.2 from docker
- Data source type & version: postgresql as shipped
- OS Grafana is installed on: linux (docker)
- User OS & Browser: safari osx
- Grafana plugins: none
- Others:

### PR DESCRIPTION
**Pull request by aknuds1**
**What this PR does / why we need it**:
Mark SQL data source macro examples as code, so they are formatted verbatim (and don't get wonky quotes).

**Which issue(s) this PR fixes**:

Fixes #23840.

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/features/datasources/postgres.md [hunk 1]
```diff
@@ -141,24 +141,24 @@ Macros can be used within a query to simplify syntax and allow for dynamic parts
...
+*`$__timeFrom()`* | Will be replaced by the start of the currently active time selection. For example, *'2017-04-21T05:01:17Z'*
+*`$__timeTo()`* | Will be replaced by the end of the currently active time selection. For example, *'2017-04-21T05:06:17Z'*
+*`$__timeGroup(dateColumn,'5m')`* | Will be replaced by an expression usable in a GROUP BY clause. For example, *(extract(epoch from dateColumn)/300)::bigint*300*
+*`$__timeGroup(dateColumn,'5m', 0)`* | Same as above but with a fill parameter so missing points in that series will be added by Grafana and 0 will be used as the value.
+*`$__timeGroup(dateColumn,'5m', NULL)`* | Same as above but NULL will be used as value for missing points.
+*`$__timeGroup(dateColumn,'5m', previous)`* | Same as above but the previous value in that series will be used as fill value. If no value has been seen yet, NULL will be used (only available in Grafana 5.3+).
+*`$__timeGroupAlias(dateColumn,'5m')`* | Will be replaced with an expression identical to $__timeGroup, but with an added column alias (only available in Grafana 5.3+).
```
Comment (oddlittlebird) [inline docs/sources/features/datasources/postgres.md:153]: I thought the problem was the quotes around the 5m?

**Comment by oddlittlebird** [comment]
I thought the issue was regarding quotes for the `5m`, that the code did not work if the quotes were included. You left the quotes around `5m` and added different ones. Does this address the issue?

**Comment by aknuds1** [comment]
> I thought the issue was regarding quotes for the `5m`, that the code did not work if the quotes were included. You left the quotes around `5m` and added different ones. Does this address the issue?

@oddlittlebird As you see from [this comment](https://github.com/grafana/grafana/issues/23840#issuecomment-619802291), the issue is that the quotes in the example get formatted so they aren't safe for copying and pasting. By enclosing the examples in [backticks](https://daringfireball.net/projects/markdown/syntax#code), they get rendered as HTML code blocks and are thus safe for copying and pasting.

**Review by oddlittlebird — APPROVED** [review]
Okeydokey.

**Comment by aknuds1** [comment]
@oddlittlebird Thanks!

**Pull request merged**